dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.connectors-2.0.feature

```diff
@@ -0,0 +1,7 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jakarta.connectors-2.0
```

Comment:
Initially I was confused as to why this was already beta. I think it is because this API is pulled in by transaction-2.0. I can't see an update to that feature to change the name?

Reply:
Hey Tom.  Thank you for catching this!!!  You are correct, I missed updating the transaction dependency on connectors-2.0.  I have updated this with an additional commit. I'll do a new build now.